js/algolia-index.js

```diff
@@ -7,6 +7,10 @@ const asciidoctor = Asciidoctor();
 const { convert } = require('html-to-text');
 const git = simpleGit(__dirname);
 
+function resolve(...args) {
```

Comment:
I'd suggest to rename this method to `resolvePath()` for a more obvious reading of the code